### ISSUE TITLE
crash XOutput

### ISSUE THREAD

**Issue by Ikaroyo** [csutorasa/XOutput#140]
i had a generic driver(without vibration feature) xoutput was working ok, installed correct driver(with vibration) and started crashing
XOutput.log:
[XOutput.log](https://github.com/csutorasa/XOutput/files/4007599/XOutput.log)
ViGEmClient.dmp as txt:
[ViGEmClient.dmp.txt](https://github.com/csutorasa/XOutput/files/4007601/ViGEmClient.dmp.txt)

**Comment by csutorasa** [comment]
Sadly this happens for a few people. As I see the proper logs are missing during the crash. I can fix the logs for the next version, and then I will have more information.

**Comment by csutorasa** [comment]
I added the new logging feature to the development release. You can try that as well and I hope the log file will contain the error.

**Comment by Ikaroyo** [comment]
ok, here the log using dev release, i notice that XOutput.log starts over at every try(erases previous entries)
[XOutput.log](https://github.com/csutorasa/XOutput/files/4008054/XOutput.log)

[ViGEmClient.dmp.txt](https://github.com/csutorasa/XOutput/files/4008055/ViGEmClient.dmp.txt)

**Comment by csutorasa** [comment]
Yes the log always resets after restart, not to waste too much space on your PC.

The log you provided is too little, so I extended it. In the newest developer version it should be better. Can you try it again?

**Comment by Ikaroyo** [comment]
There you have, log and dump
[ViGEmClient.dmp.txt](https://github.com/csutorasa/XOutput/files/4008323/ViGEmClient.dmp.txt)
[XOutput.log](https://github.com/csutorasa/XOutput/files/4008324/XOutput.log)

**Comment by csutorasa** [comment]
It is still not there, so I enabled so much error logging for developer version, that it can affect performance. Please try again, I hope for the last time.

**Comment by Ikaroyo** [comment]
its ok, every time you need it
[XOutput.log](https://github.com/csutorasa/XOutput/files/4008486/XOutput.log)
[ViGEmClient.dmp.txt](https://github.com/csutorasa/XOutput/files/4008487/ViGEmClient.dmp.txt)

**Comment by DanOwl** [comment]
> i had a generic driver(without vibration feature) xoutput was working ok, installed correct driver(with vibration) and started crashing
> if i remove the joystick XOutput doesnt crash, crashes when i reconnect the joystick(is recognized with the new drivers but crashes)
> XOutput.log:
> [XOutput.log](https://github.com/csutorasa/XOutput/files/4007599/XOutput.log)
> ViGEmClient.dmp as txt:
> [ViGEmClient.dmp.txt](https://github.com/csutorasa/XOutput/files/4007601/ViGEmClient.dmp.txt)

Just unistall your game pad drivers.

**Comment by xangeo** [comment]
its probably because you have vjoy driver as your generic gamepad driver ,to fix your issue you need to uninstall your previous generic gamepad driver and install the speedlin strike gamepad driver
link: https://drivers.softpedia.com/get/JOYSTICK-GAMEPAD-WHEELS-and-TABLETS/Speedlink/Speedlink-Strike-Gamepad-Driver-10.shtml

**Comment by PoorguyDrifting** [comment]
> It is still not there, so I enabled so much error logging for developer version, that it can affect performance. Please try again, I hope for the last time.

Hey bro i have the same issue and hoping to get a fix in next update

**Comment by franklinprakash1090** [comment]
> its probably because you have vjoy driver as your generic gamepad driver ,to fix your issue you need to uninstall your previous generic gamepad driver and install the speedlin strike gamepad driver link: https://drivers.softpedia.com/get/JOYSTICK-GAMEPAD-WHEELS-and-TABLETS/Speedlink/Speedlink-Strike-Gamepad-Driver-10.shtml

This worked for 
Zebronics zeb-260jp 
Oem: microntek USB Joystick, Shenzhen dragon rise technologies

**Comment by ahmedsamyh** [comment]
> its probably because you have vjoy driver as your generic gamepad driver ,to fix your issue you need to uninstall your previous generic gamepad driver and install the speedlin strike gamepad driver link: https://drivers.softpedia.com/get/JOYSTICK-GAMEPAD-WHEELS-and-TABLETS/Speedlink/Speedlink-Strike-Gamepad-Driver-10.shtml

Unfortunately this didn't work for me. The drivers installed but no force feedback :( i have tried to find the correct drivers for the Twin USB Joystick i have, but since i don't know the manufacturer of the joystick (i got it second-hand and its was in rough shape). the only thing i know is that it's "Blaze" Twin USB Controller. "Blaze" is written in the center of the joystick, idk if it's the manufacturers but i did search it and found some controller drivers but that didn't work too. btw i'm only using one controller of the twin joystick since the other one is broken. I cut off the wires and disabled it in device manager.

Anyways, i guess i will play without force feedback until i find another driver.
